include/revng/Pipes/PipelineManager.h

```diff
@@ -13,6 +13,7 @@
 
 #include "revng/Pipeline/Context.h"
 #include "revng/Pipeline/Runner.h"
+#include "revng/PipelineDescriptor/PipelineDescriptor.h"
```

Comment:
This include doesn't seem to be necessary since you're using a `std::string` to store the result.

Reply:
Dropped